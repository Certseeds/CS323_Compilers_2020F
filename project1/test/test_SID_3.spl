int main(){
    char 92buzai = 'a';
    return 1;
}