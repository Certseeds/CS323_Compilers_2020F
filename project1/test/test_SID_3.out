Error type A at Line 2: unknown lexeme 92buzai
